### PR TITLE
Make the token draggable

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

- Now we can drag the token in the editor!
- All the changes can be saved in the db as well

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/af4c169b-e997-415e-ac8e-3ba87648d333

### Edge Cases

1. Tested after enabled the **row** dnd and **token** dnd
2. Added new token in the active row
3. Update the column of the row

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/GalleryEditor/SortableRow.tsx [hunk 1]
```diff
@@ -247,12 +246,7 @@ export function SortableRow({
       left: 0,
       right: 0,
       zIndex: zIndex,
-      transform: [
-        { translateY: translateY.value ?? 0 },
-        {
-          scale: withSpring(isGestureActive.value ? 1.05 : 1),
-        },
-      ],
```
Comment (jakzaizzat) [inline apps/mobile/src/components/GalleryEditor/SortableRow.tsx:255]: Remove the scale for the **row** here. Kind of weird when hovering the token to drag but the **row ** as triggered too

File: apps/mobile/src/components/GalleryEditor/GalleryEditorRow.tsx [hunk 2]
```diff
@@ -42,50 +52,45 @@ export function GalleryEditorRow({ sectionId, row, style, queryRef }: Props) {
...
+          <View className="relative">
+            <SortableTokenGrid
+              columns={row.columns}
+              items={row.items}
+              size={widthPerToken}
+              scrollContentOffsetY={scrollContentOffsetY}
+              scrollViewRef={scrollViewRef}
+              onDragStart={handleDragStart}
+              onDragEnd={handleDragEnd}
+            />
```
Comment (jakzaizzat) [inline apps/mobile/src/components/GalleryEditor/GalleryEditorRow.tsx:89]: This component responsible to **render** the item in grid view

Comment (Robinnnnn) [reply apps/mobile/src/components/GalleryEditor/GalleryEditorRow.tsx:89]: can we add this as a comment?

File: apps/mobile/src/components/GalleryEditor/SortableTokenGrid/SortableTokenGrid.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  const containerHeight = useMemo(() => {
+    const rows = Math.ceil(items.length / columns);
+    return rows * (size + GAP) - GAP - rows * GAP; // Subtract GAP to remove the extra gap after the first and last row
+  }, [items.length, columns, size]);
+
+  return (
+    <View
+      style={{
+        height: containerHeight,
+      }}
```
Comment (Robinnnnn) [inline apps/mobile/src/components/GalleryEditor/SortableTokenGrid/SortableTokenGrid.tsx:55]: let's memoize this object

**Review by Robinnnnn — APPROVED** [review]
jakz you are literally a magician

File: apps/mobile/src/components/GalleryEditor/SortableTokenGrid/SortableTokenGrid.tsx (hunk 3 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/GalleryEditor/SortableTokenGrid/SortableTokenGrid.tsx]: Updated it!
